codebase/superdarn/src.lib/tk/oldfit.1.25/src/fitopen.c

```diff
@@ -112,7 +112,7 @@ struct OldFitFp *OldFitOpenFd(int fitfp,int inxfp) {
            ptr->major_rev=atoi(tmpbuf+i-j);
            for (j=0;(ptr->header[i+1+j] !=' ') && 
                (ptr->header[i+1+j] !=0);i++) tmpbuf[j]=ptr->header[i+1+j]; 
-           tmpbuf[j]=0;
+           /*tmpbuf[j]=0;*/     /* Removing this so the minor_rev is correct */
```

Comment:
Are you going to delete this line before merging so we don't have commented code?

Reply:
I hesitated to delete that line on the off chance that one day this whole git version control for the RST is abandoned and a legitimate use for that line is found.  I'm fine with it being deleted.

Reply:
why would it be abandoned?

Reply:
Well for the first ~20 years of SD RCS was used for version control - who knows what the next fad will be?  But that's just me being silly, at this point there's no reason not to just delete the line and rely on the commit history for context.

Reply:
Let me know when you delete the line then we can resolve the conversation :)